site/content/docs/developer/building-zap-with-intellij-idea.md

```diff
@@ -79,6 +79,51 @@ Note that the add-ons in _zap-extensions_ all have their own set of tasks in add
 
 ## Tips and Tricks
 
+### Connecting to a session database
+
+You can use the Database tab in IntelliJ IDEA to connect to a persisted ZAP session.
+By default, ZAP uses [HSQLDB](https://hsqldb.org/) to store its session data.
+There is also experimental support for MySQL and PostgreSQL but we won't be discussing that here.
```

Comment:
There's no experimental support for PostgreSQL (yet), better just say e.g. `for other database engines`.